src/cloud-api-adaptor/test/e2e/README.md

```diff
@@ -150,6 +150,7 @@ Use the properties on the table below for Libvirt:
 
```

Comment:
Not sure if I understand correctly what this commit brings based on the commit message. It talks about docker provider but it's modifying libvirt provider. Am I missing something? From the content I understood it adds a "container_runtime" property to be consumed by tests for the libvirt provider, isn't it?

Reply:
Yes, it's adding the new property for libvirt alike was done for docker. Sorry, it seems yet another crypt commit message that I write :)